scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create! comment_params
+    redirect_to @post
+  end
+
+  def edit
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.find(params[:id])
+  end
+
+  def update
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.update params[:id], comment_params
```

Comment:
why are you passing two arguments here? Can't you just find the `comment` from params, and update that comment by passing in `comment_params`